packages/api/src/config/Config.ts

```diff
@@ -39,6 +39,7 @@ export default class Config implements StrykerOptions {
     high: 80,
     low: 60
   };
+  public collectStatistics: string = 'yes';
```

Comment:
Je kan het type hiervan verkleinen naar een [union type](https://www.typescriptlang.org/docs/handbook/advanced-types.html#union-types) van twee strings
```suggestion
  public collectStatistics: 'yes' | 'no' = 'yes';
```